src/components/signed-out/signed-out-routes.ts

```diff
@@ -0,0 +1,18 @@
+import { PATH_NAMES } from "../../app.constants";
+
+import * as express from "express";
+import {
+  signedOutGet,
+} from "./signed-out-controller";
+import { validateSessionMiddleware } from "../../middleware/session-middleware";
+
+const router = express.Router();
+
+router.get(
+  PATH_NAMES.SIGNED_OUT,
+  validateSessionMiddleware,
```

Comment:
this is not needed as they won't have a session at this point

Reply:
removed